src/Matrix.test.js

```diff
@@ -64,4 +65,24 @@ describe("Matrix tests", () => {
 
     expect(newValue[i][j]).toEqual(v);
   });
+
+  test("addition of values", async() => {
```

Comment:
Wanna put this test in its own file (calculate.test.js). Generally it's good practice to modularize the tests such that there is a 1:1 mapping of test file to source file.  Otherwise it gets confusing as code scales (I know it doesn't matter for this case but just best practice)

Reply:
sounds good, i was wondering if i should put it in a separate file.